### ISSUE TITLE
[3.8.0-rc.1] UUID as option value is not casted as string

### ISSUE THREAD

**Issue by distantnative** [k-next/kirby#4701]
>  I had another issue, which I was able to solve but was unintuitive… I had a blueprint select using autoid plugin to define a relation (basically define a query and use {{ page.autoid }} as the value). Changing value to page.uuid didn't work, I had to change to page.uuid.id…  I would expect the "toString" of an UUID field to compute to the actual UUID, without needing to access it via id… (if I'm understanding this correctly)

**Comment by distantnative** [comment]
Tried to reproduce, but this does work for me

```yaml
testy:
  type: select
  options:
    type: query
    query: site.children
    value: "{{ page.uuid }}"
```

**Comment by afbora** [comment]
I can't reproduce either.

**Comment by doup** [comment]
It was me that reported this on Discord. After your comments, I just noticed that the issue wasn't that, the dropdown gets populated correctly. The thing is that when migrating from `autoid` I just changed the old set of ids for the new ones:

```diff
- Categories: autoidID
+ Categories: kirbyUUID
```

Toying around I noticed that `"{{ page.uuid }}"` gets saved as `page://kirbyUUID`, and `"{{ page.uuid.id }}"` as `kirbyUUID`; that's why I though `{{ page.uuid.id }}` was needed as that worked with my migrations.

So, `page://kirbyUUID` is the prefered way?
